venues/auai.org/UAI/2018/python/assign-all-reviewers.py

```diff
@@ -27,7 +27,11 @@
 
         for entry in assignment_entries:
             openreview.tools.assign(client, paper.number, 'auai.org/UAI/2018',
-                parent_group_params = {},
+                individual_group_params = {'readers': [
```

Comment:
Just to check, are you adding the author group as nonreaders

Reply:
yes!

https://openreview.net/groups?id=auai.org/UAI/2018/Paper220/AnonReviewer1

Reply:
also you can see here:

https://github.com/iesl/openreview-py/blob/master/openreview/tools.py#L390